tools/src/test/java/org/apache/kafka/tools/TopicCommandIntegrationTest.java

```diff
@@ -85,653 +80,761 @@
 import static org.mockito.Mockito.spy;
 
 @Tag("integration")
```

Comment:
Could you please remove this tag? it is redundant to `ClusterTestExtensions`, as it is added automatically.